front/app/components/Form/Components/Controls/InputControl.tsx

```diff
@@ -47,7 +46,7 @@ export const InputControl = ({
   const FieldLabel = () => {
     return (
       <Box display="flex">
-        <Text>{label}</Text>
+        {label}
         {uischema?.options?.isAdminField && (
```

Comment:
Styling is handled by the FieldLabel component, so we shouldn't have a <Text> component here.